src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3040,6 +3040,37 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <ReferenceComWrappersToCopyLocal Include="@(ComReferenceWrappers)" Condition="'%(ComReferenceWrappers.CopyLocal)'!='false'"/>
     </ItemGroup>
 
+  </Target>
+
+    <!--
+    ============================================================
+                                        GetAssembliesMetadata
+
+    Resolve Assembly attributes for COM assemblies
```

Comment:
nit: Maybe remove "COM" from this comment and add a note that the target is intended to be used only by Visual Studio?